src/transcendental.rs

```diff
@@ -18,38 +18,38 @@
 
```

Comment:
Is it accurate to say that the changes to transcendental.rs should not affect lbp calculations, i.e. the changes to existing functions we currently use should not affect their output?
I ask because there was lots of rounding testing done for LBP so I would be a little alarmed if we are changing types in a way that affects output of existing functions... but it looks like we aren't?

Reply:
Yes, the changes should not affect lbp (or any other usage of the previously existing math functions here) at all. This only adds the new `saturating_powi_high_precision` function.